dist/tpm2-abrmd.service.in

```diff
@@ -1,12 +1,12 @@
-# copy this file into /etc/systemd/system
+# copy this file into /lib/systemd/system
```

Comment:
I would just remove this part since where things get copied really depend on the configure flags used.

Reply:
Accepted.